code/modules/events/blob/theblob.dm

```diff
@@ -268,6 +268,10 @@ GLOBAL_LIST_EMPTY(blob_nodes)
 			return B.blob_reagent_datum.description
 	return "something unknown"
 
+/obj/structure/blob/carbon_throw_hit(mob/living/carbon/human/C, datum/thrownthing/throwingdatum, damage, mob_hurt, self_hurt)
+	damage *= 0.25 // Lets not have sorium be too much of a blender / rapidly kill itself
```

Comment:
is this intended to stack with the `obj` level multiplication of 0.75?

Reply:
Yes. More for the knockdown over damage if it goes into another blob, otherwise it will rip itself apart